pkg/bot/mattermost.go

```diff
@@ -2,17 +2,30 @@ package bot
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"net/url"
 	"regexp"
+	"strconv"
 	"strings"
+	"sync"
 
 	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/sirupsen/logrus"
 
 	"github.com/kubeshop/botkube/pkg/config"
+	"github.com/kubeshop/botkube/pkg/events"
+	format2 "github.com/kubeshop/botkube/pkg/format"
+	"github.com/kubeshop/botkube/pkg/multierror"
 )
 
+// TODO: Refactor:
+// 	- handle and send methods from `mattermostMessage` should be defined on Bot level,
+//  - split to multiple files in a separate package,
+//  - review all the methods and see if they can be simplified.
```

Comment:
```suggestion
// TODO: Refactor:
//    - handle and send methods from `mattermostMessage` should be defined on Bot level,
//    - split to multiple files in a separate package,
//    - review all the methods and see if they can be simplified.
```

BTW, are we going to have a single issue related to all those refactor actives? I can be just a single issue that captures all `TODO: Refactor:` comments that you added.

Reply:
Good idea - added issue https://github.com/kubeshop/botkube/issues/667. Will refer that in the code.